lib/identity_cache/configuration_dsl.rb

```diff
@@ -236,7 +236,11 @@ def #{options[:cached_accessor_name]}
             if IdentityCache.should_use_cache? || #{association}.loaded?
               @#{options[:records_variable_name]} ||= #{options[:association_reflection].klass}.fetch_multi(#{options[:cached_ids_name]})
             else
-              #{association}
+              if IdentityCache.fetch_returns_relation
+                #{association}
+              else
+                association(:#{association}).load_target
```

Comment:
I think `#{association}.to_a` would be preferred since that is part of the documented active record API.